src/Command/Mute.js

```diff
@@ -0,0 +1,117 @@
+const Command = require('../Command');
+
+module.exports = Command.extend({
+    commandName: 'mute',
+    commandAliases: ['ban'],
+    advertisable: false,
+    processMessage: function (message, tokens) {
+        let hasRoles = message.member !== null && typeof message.member.roles !== 'undefined';
+
+        if (hasRoles && this.memberIsAdministrator(message.member)) {
+            message.delete();
+
+            let victim = message.mentions.members.first();
+
+            if (tokens.length < 3 || !victim) {
+                message.member.user.send('Please use the correct format: `' + tokens[0] + ' <@User#9999> <Reason>`.');
+            }
+            else if (message.member.toString() == victim.toString()) {
+                message.member.user.send('You may not mute yourself!');
+            }
+            else if (this.memberIsAdministrator(victim)) {
+                message.member.user.send('You may not mute another Discord administrator!');
+            }
+            else {
+                let reason = tokens.slice(2).join(' ');
+
+                if (!reason || reason == '' || reason == null) {
+                    message.member.user.send('Please use the correct format: `' + tokens[0] + ' <@User#9999> <Reason>`.');
+                }
+                else {
+                    message.member.user.send({
+                        embed: {
+                            color: 0xff0000,
+                            author: {
+                                name: this.discordClient.user.username,
+                                icon_url: this.discordClient.user.avatarURL
+                            },
+                            title: "Successfully muted @" + victim.user.username + "#" + victim.user.discriminator,
+                            fields: [
+                                {
+                                    name: "Mute Reason",
+                                    value: reason
+                                }
+                            ],
+                            timestamp: new Date(),
+                            footer: {
+                                icon_url: this.discordClient.user.avatarURL,
+                                text: "Shotbow Network Chat Bot"
```

Comment:
"Shotbow Chat Bot" please - in all places

Reply:
I am ashamed that I actually did this